### PR TITLE
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by brittmsantos** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@bfjelds @msalehmsft @parameshbabu

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Comment by alexbuckgit** [comment]
@bfjelds @cmcclister @DhurataJ @GrantMeStrength @Khairunj @mattwojo @msalehmsft @parameshbabu @rsameser @v-hearya @v-pmehrabian

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

File: README.md [hunk 1]
```diff
@@ -23,15 +23,14 @@ or trademarks, whether by implication, estoppel or otherwise.
 
 ## Contributing
 
-This is the repository for Windows 10 IoT **documentation** hosted at [https://docs.microsoft.com/windows/iot-core](https://docs.microsoft.com/windows/iot-core).
+This is the repository for published [Windows for IoT documentation](https://docs.microsoft.com/windows/iot-core)(.
 
-If you would like to see new coverage or have feedback, please consider [**contributing**](/CONTRIBUTING.md).  You can edit the existing content, add new content, or simply create new [issues](https://github.com/MicrosoftDocs/windows-iotcore-docs/issues). We’ll take a look at your suggestions and will work together to incorporate them into the docs.
+If you would like to see new coverage or have feedback, please consider [**contributing**](https://docs.microsoft.com/CONTRIBUTING).  You can edit the existing content, add new content, or simply create new [issues](https://github.com/MicrosoftDocs/windows-iotcore-docs/issues). We’ll take a look at your suggestions and will work together to incorporate them into the docs.
```
Comment (parameshbabu) [inline README.md:28]: Does this page exist? i dont think this path is correct. It is supposed to link to the Contributing.md file in the same directory.

Comment (alexbuckgit) [reply README.md]: Fixed, thanks for the feedback.

**Comment by alexbuckgit** [comment]
@parameshbabu - The flagged issue has been resolved.

Please merge, thanks!

**Review by parameshbabu — APPROVED** [review]

**Review by parameshbabu — APPROVED** [review]

**Pull request merged**